m4/angry_bird.py

```
@@ -0,0 +1,241 @@
+        self.launch_angle = 0
+        self.game_over = False
+        
+    def handle_mouse_events(self, mouse_pos, mouse_pressed):
+        if len(self.birds) > 0 and not self.birds[0].launched:
+            bird = self.birds[0]
+            
+            if mouse_pressed:
+                # Bug 2: 数学计算错误 - 角度计算有误
+                dx = mouse_pos[0] - bird.x
+                dy = mouse_pos[1] - bird.y
+                self.launch_angle = math.atan2(dy, dx)  # 错误：应该考虑方向
```

Ruff (0.13.1)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)